kover-jvm-agent/src/main/java/kotlinx/kover/jvmagent/KoverAgentSettings.java

```diff
@@ -0,0 +1,11 @@
+package kotlinx.kover.jvmagent;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KoverAgentSettings {
```

Comment:
It seems that this class does not need equals/hashCode, but maybe add `.toString()` for better debuggability?